modules/@apostrophecms/util/ui/public/2-http.js

```diff
@@ -127,7 +129,9 @@
         } else {
           query = {};
         }
-        query['apos-mode'] = 'draft';
+        if (typeof options.draft !== 'undefined') {
```

Comment:
Use `options.draft != null` instead (use of only one = is intentional). That's consistent with our "null means don't care" policy elsewhere and it also matches undefined and saves code. Sorry

Reply:
this entire block it wrapped in a `if (options.draft != null) {` ... I must be really missing something here